<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArbPower</class>
 <widget class="QMainWindow" name="ArbPower">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArbPower</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Res/interstate.png</normaloff>:/Res/interstate.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="baseEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>79</y>
      <width>112</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="whatsThis">
     <string>底数</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Res/kb.png);</string>
    </property>
    <property name="text">
     <string>12.34567</string>
    </property>
   </widget>
   <widget class="QLabel" name="powerLabel">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>78</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exponentEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>79</y>
      <width>112</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="whatsThis">
     <string>指数</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Res/kb.png);</string>
    </property>
    <property name="text">
     <string>2.45678</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerShow">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>21</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Res/kb.png);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>139</y>
      <width>242</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/Res/button_calculate_normal.png);</string>
    </property>
    <property name="text">
     <string>计  算</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerShow_2">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>69</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Res/kb.png);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerShow_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>69</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Res/kb.png);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>2</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>系统函数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>51</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>泰勒展开方法</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>51</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>泰勒 - 科学计数法</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>111</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="baseSignLabel">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>111</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="exponentSignLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>23</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×10</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerExponentlabel">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>19</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerExponentlabel_2">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>66</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×10</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerExponentlabel_3">
    <property name="geometry">
     <rect>
      <x>808</x>
      <y>64</y>
      <width>180</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>781</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>245</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>结果显示精度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>99</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>常微分方程方法</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerShow_4">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>117</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Res/kb.png);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>118</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×10</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerExponentlabel_4">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>114</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="prec_comboBox">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>242</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Res/kb.png);</string>
    </property>
    <item>
     <property name="text">
      <string>10</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/Res/ten.png</normaloff>:/Res/ten.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/Res/fourteen.png</normaloff>:/Res/fourteen.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/Res/eightteen.png</normaloff>:/Res/eightteen.png</iconset>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>99</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>常微分方程 - 科学计数法</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>781</x>
      <y>118</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×10</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerExponentlabel_5">
    <property name="geometry">
     <rect>
      <x>808</x>
      <y>114</y>
      <width>180</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerShow_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>117</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Res/kb.png);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerShow_6">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>165</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Res/kb.png);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>781</x>
      <y>166</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerShow_7">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>165</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Res/kb.png);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>166</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>147</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>数值微分 - 科学计数法</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerExponentlabel_6">
    <property name="geometry">
     <rect>
      <x>808</x>
      <y>162</y>
      <width>180</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>147</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>数值微分-龙贝格算法</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerExponentlabel_7">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>162</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>214</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerShow_8">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>213</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Res/kb.png);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>195</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>方程求根</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerExponentlabel_8">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerShow_9">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>213</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Res/kb.png);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>781</x>
      <y>214</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×10</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerExponentlabel_9">
    <property name="geometry">
     <rect>
      <x>808</x>
      <y>209</y>
      <width>180</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>195</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>方程求根 - 科学计数法</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>显示精度(&amp;P)</string>
    </property>
    <addaction name="action10"/>
    <addaction name="action14"/>
    <addaction name="action18"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <action name="action10">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/ten.png</normaloff>:/Res/ten.png</iconset>
   </property>
   <property name="text">
    <string>10位(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>10位有效数字</string>
   </property>
  </action>
  <action name="action14">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/fourteen.png</normaloff>:/Res/fourteen.png</iconset>
   </property>
   <property name="text">
    <string>14位(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>14位有效数字</string>
   </property>
  </action>
  <action name="action18">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/eightteen.png</normaloff>:/Res/eightteen.png</iconset>
   </property>
   <property name="text">
    <string>18位(&amp;E)</string>
   </property>
   <property name="toolTip">
    <string>18位有效数字</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
